app/src/main/java/com/codesoom/assignment/domain/Tasks.java

```diff
@@ -0,0 +1,76 @@
+package com.codesoom.assignment.domain;
+
+import java.util.*;
+
+/**
+ * {@code Tasks} is the manager of task
+ *
+ * @author etff
+ * @version 1.0.0 21/02/29
+ * @see com.codesoom.assignment.domain.Task
+ */
+public class Tasks {
+    private Map<Long, Task> tasks;
+    private Long taskId = 1L;
+
+    public Tasks() {
+        this.tasks = new LinkedHashMap<>();
+    }
+
+    /**
+     * Add task to tasks
+     *
+     * @param task item of task
+     */
+    public void addTask(Task task) {
+        Long generateId = generateId();
+        task.setId(generateId);
+        tasks.put(generateId, task);
+    }
+
+    /**
+     * Remove task
+     *
+     * @param id target id of task
+     */
+    public void remove(Long id) {
+        this.tasks.remove(id);
+    }
+
+    /**
+     * Find target task
```

Comment:
JavaDoc을 작성할 때에는 리턴값에 대한 설명이 있는 것이 좋습니다. `대상 할 일을 찾는다`보다 `넘겨받은 아이디에 해당하는 할 일을 찾아 리턴한다`는 설명이 조금 더 자세하죠. 이 메소드는 단순하지만 코딩을 하다 보면 도대체 이 메소드가 리턴하는 값의 의미가 뭐야? 싶을 때가 많습니다. 주석으로 그 문제를 어느 정도 해결할 수도 있죠.

```suggestion
     * 주어진 아이디에 해당되는 할 일을 찾아 리턴합니다.
```

Reply:
👍👍👍👍👍